ERROR - 2022-07-18 17:12:01 --> Error connecting to the database: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente.

CRITICAL - 2022-07-18 17:12:01 --> Unable to connect to the database.
Main connection [MySQLi]: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente.

#0 C:\xampp\htdocs\food\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\food\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\food\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\food\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\food\app\Models\UsuarioModel.php(113): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\food\app\Libraries\Autenticacao.php(26): App\Models\UsuarioModel->buscaUsuarioPorEmail('antonio@antonio...')
#6 C:\xampp\htdocs\food\app\Controllers\Login.php(31): App\Libraries\Autenticacao->login('antonio@antonio...', '1234')
#7 C:\xampp\htdocs\food\system\CodeIgniter.php(832): App\Controllers\Login->criar()
#8 C:\xampp\htdocs\food\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\food\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\food\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\food\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#12 {main}
